config/certificates_config.go

```diff
@@ -0,0 +1,31 @@
+package config
+
+import "os"
+
+type CertificatesConfig struct {
+	CertBoletoPathCrt             string
+	CertBoletoPathKey             string
+	CertBoletoPathCa              string
+	CertICP_PathPkey              string
+	CertICP_PathChainCertificates string
+	CertificatesPath              string
+	CertificateICPName            string
+	PswCertificateICP             string
+	CertificateSSLName            string
+	PswCertificateSSL             string
+}
+
+func getCertificatesConfig() CertificatesConfig {
```

Comment:
Opa, sem voltar o ponteiro, tu vai ter criado um objeto(struct), e voltar uma cópia dele 😀

```suggestion
func getCertificatesConfig() *CertificatesConfig {
```